src/main/java/juja/sqlcmd/view/Console.java

```diff
@@ -0,0 +1,22 @@
+package juja.sqlcmd.view;
+
+import java.util.Scanner;
+
+public class Console implements View {
+    Scanner scanner;
```

Comment:
Perhaps we can make the Scanner scanner class field private

Reply:
Yep)